Sources/CGFloatExtensions.swift

```diff
@@ -47,5 +47,14 @@ extension CGFloat {
     public static func radiansToDegrees(_ angleInDegrees: CGFloat) -> CGFloat {
         return (180.0 * angleInDegrees) / .pi
     }
-
+    
+    /// EZSE: Returns a random floating point number between 0.0 and 1.0, inclusive.
+    public static func random() -> CGFloat {
```

Comment:
@piv199 What if we deprecate this one and create a new? We already did this on some other methods:
``` swift
    /// EZSE: Checks if array contains at least 1 instance of the given object type
    @available(*, deprecated: 1.8, renamed: "containsType(of:)")
    public func containsInstanceOf<T>(_ element: T) -> Bool {
        return containsType(of: element)
    }

    /// EZSE: Gets the object at the specified index, if it exists.
    @available(*, deprecated: 1.8, renamed: "get(at:)")
    public func get(_ index: Int) -> Element? {
        return get(at: index)
    }

    /// EZSE: Checks if all elements in the array are true of false
    @available(*, deprecated: 1.8, renamed: "testAll(is:)")
    public func testIfAllIs(_ condition: Bool) -> Bool {
        return testAll(is: condition)
    }
```